pkg/build/registry/buildlog/rest.go

```diff
@@ -38,7 +38,7 @@ type REST struct {
 	getSimpleLogsFn func(podNamespace, podName string, logOpts *kapi.PodLogOptions) (runtime.Object, error)
 }
 
-const defaultTimeout time.Duration = 10 * time.Second
+const defaultTimeout time.Duration = 60 * time.Second
```

Comment:
let's godoc what this timeout is (it is how long we wait for the build to be running before giving up on getting logs for a build)